settings.py

```diff
@@ -35,7 +35,7 @@
     DATABASES = {
         'default': {
             'ENGINE': 'django.db.backends.sqlite3',
-            'NAME': '/Users/gatana/wger/wg-tomatoes/database.sqlite',
+            'NAME': '/Users/kamaradeo/Desktop/Travail/wg-tomatoes/database.sqlite',
```

Comment:
Hey @dkam26 Please change this back to the agreed on `./database.sqlite` value and don't push your new Settings.py as part of the PR, as agreed

Reply:
Hey Joan,I am on it